src/main/java/com/audit/controller/AuditController.java

```diff
@@ -102,81 +99,102 @@
 			request.getSession(true);
 			return new ResponseEntity<String>(gson.toJson("Login successful"), HttpStatus.OK);
 		} else {
-			return new ResponseEntity<String>(gson.toJson("Username and/or Password incorrect"),
-					HttpStatus.NOT_ACCEPTABLE);
```

Comment:
how do we handle this not acceptable status code to ui

Reply:
Its handled in CommonExceptionHandler.java